web/src/main/AllCreatedTopicsBox.js

```diff
@@ -0,0 +1,147 @@
+    this.state = {
+      classes: props,
+      memberId: props.match.params.memberId,
+      tab: props.match.params.tab,
+      topics: [],
+      TAB_LIST: [
+        {label: "Q&A", value: "qna"},
+        {label: "Tech", value: "tech"},
+        {label: "Play", value: "play"},
+        {label: "Jobs", value: "jobs"},
+        {label: "Deals", value: "deals"},
+        {label: "City", value: "city"}
+      ]
+    };
+  }
+
+  componentDidMount() {
+      this.getAllCreatedTopics();
+  }
+
+  getAllCreatedTopics() {
+    TopicBackend.getAllCreatedTopics(this.state.memberId, this.state.tab, 10, 1)
+      .then((res) => {
+        this.setState({
+          topics: res,
+        });
+      });
+  }
+
+  renderTab(tab) {
+    return (
+      {
+        ...this.state.tab === tab.value ?
+          <a href={`/member/${this.state.memberId}/${tab.value}`} class="cell_tab_current"> {tab.label} </a> :
+          <a href={`/member/${this.state.memberId}/${tab.value}`} class="cell_tab"> {tab.label} </a>
+      }
+    )
+  }
+
+  renderTopic(topic) {
+    const style = topic.nodeId !== "promotions" ? null : {
+      backgroundImage: `url('${Setting.getStatic("/static/img/corner_star.png")}')`,
+      backgroundRepeat: "no-repeat",
+      backgroundSize: "20px 20px",
+      backgroundPosition: "right top"
+    };
+
+    return (
+      <div className="cell item" style={style}>
+        <table cellPadding="0" cellSpacing="0" border="0" width="100%">
+          <tbody>
+          <tr>
+            <td width="auto" valign="middle">
+              <span className="item_title">
+                <a href={`/t/${topic.id}`} className="topic-link"> {topic.title} </a>
+              </span>
+              <div className="sep5" />
+              <span className="topic_info">
+                <div className="votes" />
+                <a className="node" href={`/go/${topic.nodeId}`}> {topic.nodeName} </a>
+                &nbsp;•&nbsp;
+                <strong><a href={`/member/${topic.author}`}> {topic.author} </a></strong>
+                &nbsp;•&nbsp;
+                {Setting.getPrettyDate(topic.createdTime)}
+                &nbsp;•&nbsp;
+                last reply from
+                <strong>
+                  <a href={`/member/${topic.lastReplyUser}`}> {topic.lastReplyUser} </a>
+                </strong>
+              </span>
+            </td>
+              <td width="70" align="right" valign="middle">
+                <a href={`/t/${topic.id}`} className="count_livid">6</a>
+              </td>
+          </tr>
+          </tbody>
+        </table>
+      </div>
+    )
+  }
+
+  render() {
+    {
+      if (this.state.tab === "replies") {
+        return (
+          <div />
+        );
+      }
+    }
+    return (
+      <div className="box">
+        <div class="cell_tabs">
+          <div class="fl">
+            <img src={Setting.getUserAvatar(this.state.memberId)} width={24} border={0} style={{"border-radius": "24px", "margin-top": "-2px"}}/>
```

Comment:
Change to `{{borderRadius: "24px", marginTop: "-2px"}}`, no double-quote in the map key, the variable also needs to renamed from underline to camel. Same to other styles.